components/chart/src/heatfield/chart-heatfield.ts

```diff
@@ -201,30 +188,44 @@ export class DtChartHeatfield extends _DtHeatfieldMixinBase
   @ViewChild(CdkConnectedOverlay, { static: false })
   _overlay: CdkConnectedOverlay;
 
+  private _destroy$ = new Subject<void>();
+  private _relativeBoundingBox: { top: number; left: number; width: number };
+
+  /**
+   * Highcharts charts object is needed for the xAxis is
+   * set by the chart via the `_initHeatfield` function that
+   * is called by the chart.
+   */
+  private _chartObject: Highcharts.ChartObject;
+
   constructor(
     elementRef: ElementRef,
-    // tslint:disable-next-line: no-forward-ref
```

Comment:
You are removing parameters from the constructor, which in this instance is considered a breaking change. 
Please change the commit message to a `fix` component and add a `Breaking Change` codename.